mne_icalabel/gui/_label_components.py

```diff
@@ -1,7 +1,7 @@
 from typing import Dict, List, Union
 
 from matplotlib import pyplot as plt
-from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg
+from matplotlib.backends.backend_qtagg import FigureCanvasQTAgg
```

Comment:
And that is why we should test with all 4 Qt bindings like the Qt-browser 😓

Reply:
What happened?

Reply:
IIRC, it was failing on the Linux CI when I changed the installation from 'qt5-defaut' to 'qt6-base-dev'.
I'm guessing the backend 'qt_agg' is compatible with all bindings, while 'qt5_agg' works only with Qt5.